File: src/com/brianc/css/Rule.java
package com.brianc.css;

import java.util.List;

public class Rule {
	public List<Selector> getSelectors() {
		return selectors;
	}

	public List<Declaration> getDeclarations() {
		return declarations;
	}

	List<Selector> selectors;
	List<Declaration> declarations;

	public Rule(List<Selector> selectors, List<Declaration> declarations) {
		this.selectors = selectors;
		this.declarations = declarations;
	}
}

File: src/com/brianc/css/Rule.kt
package com.brianc.css

class Rule(selectors: List<Selector>, declarations: List<Declaration>) {

    var selectors: List<Selector>
        internal set
    var declarations: List<Declaration>
        internal set

    init {
        this.selectors = selectors
        this.declarations = declarations
    }
}
